intranet/apps/eighth/tests/admin/test_admin_groups.py

```diff
@@ -531,23 +531,23 @@ def test_delete_empty_groups(self):
         self.make_admin()
 
         # Create users
-        userA = get_user_model().objects.get_or_create(username="userA")[0]
+        users = get_user_model().objects.get_or_create(username="users")[0]
         userB = get_user_model().objects.get_or_create(username="userB")[0]
```

Comment:
Oops

Reply:
Reverted this change, it shouldn't be here :)